networkx/algorithms/tests/test_cycles.py

```diff
@@ -886,6 +886,7 @@ def test_tree_graph(self):
         (nx.heawood_graph(), 6),
         (nx.pappus_graph(), 6),
         (nx.random_tree(10, seed=42), inf),
+        (nx.empty_graph(10), inf),
```

Comment:
Adding the test from @boothby 's version
```suggestion
        (nx.empty_graph(10), inf),
        (nx.Graph(chain(cycle_edges(range(5)), cycle_edges(range(6, 10)))), 4),
```